scss/partials/_cards.scss

```diff
@@ -160,7 +171,13 @@
 
   &:hover {
     .content-block__title {
-      @extend .a;
+      color: $primary-blue;
+      --d: 100%;
```

Comment:
CSS variables!

Reply:
Is it a no-no? The `anchor--*` class is built with this method and I'm basically just copy it here. Maybe we can do a refactor post mvp